web app/src/test/java/de/tinf15b4/kino/data/SearchServiceTest.java

```diff
@@ -0,0 +1,77 @@
+package de.tinf15b4.kino.data;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Locale;
+import java.util.Random;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.github.javafaker.Faker;
+
+import de.tinf15b4.kino.data.cinemas.Cinema;
+import de.tinf15b4.kino.data.cinemas.CinemaService;
+import de.tinf15b4.kino.data.movies.AgeControl;
+import de.tinf15b4.kino.data.movies.Genre;
+import de.tinf15b4.kino.data.movies.Movie;
+import de.tinf15b4.kino.data.movies.MovieService;
+import de.tinf15b4.kino.data.search.SearchResult;
+import de.tinf15b4.kino.data.search.SearchService;
+import de.tinf15b4.kino.web.KinoWebApplication;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = { KinoWebApplication.class })
+public class SearchServiceTest {
+    @Autowired
+    private SearchService searchService;
+
+    @Autowired
+    private MovieService movieRepo;
+
+    @Autowired
+    private CinemaService cineRepo;
```

Comment:
I dont like using real repositories as its not the repositories we are testing here. Try mocking them. Injects should never be needed in unit tests if possible (and it is possible here)